src/test/java/TestResultsParserTests.java

```diff
@@ -16,6 +15,32 @@ void getSomeResult() {
         assertTrue(testResults.size() > 0);
     }
 
+    @Test
+    void noResultsWithoutBuild() {
+        List<TestResultsParser.TestResult> testResults = TestResultsParser.getResults();
+        assertNotNull(testResults);
+        assertEquals(0, testResults.size());
+    }
+
+    @Test
+    void canParseresultsXML() {
+        createDummyTestResults();
+        List<TestResultsParser.TestResult> testResults = TestResultsParser.getResults();
+        TestResultsParser.TestResult result = testResults.get(0);
+
+        assertEquals(result.testSuiteName, "HelloWorldTest");
+        assertEquals(result.testsCount, 2);
```

Comment:
We talked about having one assert per test, so perhaps you should change that?

Reply:
It's more about testing one thing per test, and this test verifies that all the XML values can be parsed, which is the result of one function called once with one input.

Reply:
I see, then it should be fine!